frontend/src/components/common/Navbar/Navbar.less

```
@@ -47,3 +47,11 @@ body.light .navbar-button:focus {
 body.light .navbar-title {
   color: @button-light-hover;
 }
+
+.cl-navbar {
+  background-color: #141414;
+
+  body.light {
+    background-color: white;
+  }
+}
```

This approach seems much more effective to me. However I'm not convinced it could work, shouldn't it be like this?
```suggestion
.cl-navbar {
  background-color: #141414;
}
body.light .cl-navbar {
    background-color: white;
  }

```

---

your suggestion is right even if don't change the final result. Outside this suggestion we've noted this change (from hard-coded style to dynamic css classes) produces some problems to the "trasparence effect " of the drawer cause ANT has a very strange behavior. So we'd know if it's a good idea to apply this change breaking the right behavior of the drawer. Unfortunately we think that there isn't a good solution for everything: we've to decide if accept a bad hard code or a good code with some problems